sdk/core/core-rest-pipeline/vitest.browser.config.mts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { defineConfig } from "vitest/config";
```

Comment:
We should probably remove karma.conf as well right? Or do we need to keep it around?

Reply:
yeah we should absolutely remove it 👍

Reply:
removed!